static/js/reducers/channel_contributors.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import { GET, INITIAL_STATE } from "redux-hammock/constants"
+
+import type { ChannelContributors } from "../flow/discussionTypes"
```

Comment:
small thing, but can you move the type import below the `../lib/api` import (and put a blank line in-between)